### ISSUE TITLE
No option to disable connection bar in RDP

### ISSUE THREAD

**Issue by bpsib** [mRemoteNG/mRemoteNG#1663]
I can't find an option to disable the RDP connection bar from showing when moving the mouse cursor to the top while it's full screen.

## Your Environment

* Version used: 1.76.20
* Operating System and version (e.g. Windows 10 1709 x64):
Windows 10 1903 x64

**Comment by dimuththarindu** [comment]
I'm having a similar issue. I would also like an option to disable it or hide it properly (CTRL-ALT-HOME).

**Comment by xbliss** [comment]
I agree and third this setting. I hate that it keeps popping up especially when I have tons of Browser Tabs open on the Host machine. Keep having to drag it around. Wish it could go to the SIDES or BOTTOM or something/ somewhere else or be DISABLED as was the case in RDCMan.

**Comment by VShawn** [comment]
@bpsib @dimuththarindu @xbliss 
Hi guys, welcome you to try my new app to fix this little issue.

https://github.com/VShawn/PRemoteM

![Snipaste_2020-12-01_17-50-10](https://user-images.githubusercontent.com/10143738/100724448-db110400-33fd-11eb-97b3-a369dca6b6d3.jpg)

**Comment by themegabyte** [comment]
Hey All,

Just bumping this if this is still being considered.

I believe we are trying to replicate this:

![image](https://user-images.githubusercontent.com/9980686/178735981-7c35bd3f-c2fc-4fed-8df7-86726d7895f2.png)
